sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -220,6 +220,12 @@
    */
   private @Nullable Boolean enableUncaughtExceptionHandler = true;
 
+  /*
+   * When enabled, UncaughtExceptionHandler will print exceptions (same as java would normally do),
+   * if no other UncaughtExceptionHandler was registered in the thread before.
```

Comment:
```suggestion
   * if no other UncaughtExceptionHandler was registered before.
```

Removing `in the thread` before because we use `setDefaultUncaughtExceptionHandler` and this triggers for all threads.